src/modules/utils/Utils.ts

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+
+    Includes various useful functions
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import { UInt64 } from "spu-integer-math";
+
+export class Utils
+{
+    /**
+     * Check whether the string is a number.
+     * @param value
+     */
+    public static isNumber (value: string): boolean
+    {
+        return /^[+\-]?([0-9]+)$/.test(value);
+    }
+
+    /**
+     * Check whether the string is a positive number.
+     * @param value
+     */
+    public static isPositiveNumber (value: string): boolean
+    {
+        return /^(\+)?([0-9]+)$/.test(value);
+    }
+
+    /**
+     * Check whether the string is a negative number.
+     * @param value
+     */
+    public static isNegativeNumber (value: string): boolean
+    {
+        return /^\-([0-9]+)$/.test(value);
+    }
+
+    /**
+     * Converts the UInt64 to the string
+     * @param value The number of UInt64
+     * @param radix An integer in the range 2 through 36 specifying
+     * the base to use for representing numeric values.
+     */
+    public static UInt64ToString(value: UInt64, radix?: number): string
```

Comment:
Are you planning to add a reverse converter? (StringToUInt64)

Reply:
It is already embedded in UByte64.